rocker-compiler/src/test/java/com/fizzed/rocker/compiler/TemplateParserTest.java

```diff
@@ -829,6 +829,60 @@ public void testSwitchBlockWithoutDefault() throws Exception {
         assertThat(case2.getExpression(), is("\"test2\""));
 
 
+    }
```

Comment:
Would it be possible for you to add an example of comma-delimited cases as 1 more unit test?  Perhaps it'd be good to also use something like an Enum as well.  Maybe include a simple sample Enum we utilize in a template and have one of the cases match multiple enums with commas?

Reply:
im not sure what you mean by comma-delimited - can you elaborate?
also the enum test should be in rocker-test-template17 right ? (since you mean to check the actual output of the template generation)

Reply:
done , let me know if this is ok